### PR TITLE
⬆️ Upgrade traefik to major version 3

### PR DESCRIPTION
**Pull request by mrnicegyu11**
## What do these changes do?
- Upgrades traefik to v3
- For now uses traefik-v2 style routing (this needs to be changed eventually...)
- Change: `docker` provider is now called `swarm`
- Change: Already prepare OTLP tracing and remove old-style jaeger/zipkin connection

## Related PRs
- https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/merge_requests/770
- https://github.com/ITISFoundation/osparc-ops-environments/pull/731

## Dev-ops checklist

- [x] No ENV changes or I properly updated ENV ([read the instruction](https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/blob/configs/README.md?ref_type=heads#how-to-update-env-variables))

### CODE REVIEW

File: services/docker-compose.yml [hunk 1]
```diff
@@ -1182,17 +1182,18 @@ services:
...
-      - "--tracing.jaeger.localAgentHostPort=jaeger:6831"
+      - "--providers.swarm.refreshSeconds=1"
+      - "--providers.swarm.exposedByDefault=false"
+      - "--providers.swarm.constraints=Label(`io.simcore.zone`, `${TRAEFIK_SIMCORE_ZONE}`)"
+      - "--core.defaultRuleSyntax=v2"
+      - "--tracing"
+      - "--tracing.addinternals"
+      - "--tracing.otlp=true"
+      - "--tracing.otlp.http=true"
+      # - "--tracing.otlp.http.endpoint=0.0.0.0:4318/v1/traces"
```
Comment (pcrespov) [inline services/docker-compose.yml:1196]: please rm comments if not used.

**Review by pcrespov — APPROVED** [review]
thx

**Review by sanderegg — APPROVED** [review]
all good thanks!

**Review by YuryHrytsuk — APPROVED** [review]

**Review by YuryHrytsuk — APPROVED** [review]

File: services/docker-compose.yml (hunk 1 above)
Comment (mrnicegyu11) [reply services/docker-compose.yml:1196]: Good catch thx

**Pull request merged**